pkg/cli/admin/upgrade/recommend/recommend.go

```diff
@@ -145,82 +138,138 @@ func (o *options) Run(ctx context.Context) error {
 		}
 	}
 
-	if len(cv.Status.AvailableUpdates) > 0 {
-		fmt.Fprintf(o.Out, "\nRecommended updates:\n\n")
-		// set the minimal cell width to 14 to have a larger space between the columns for shorter versions
-		w := tabwriter.NewWriter(o.Out, 14, 2, 1, ' ', 0)
-		fmt.Fprintf(w, "  VERSION\tIMAGE\n")
-		// TODO: add metadata about version
-		sortReleasesBySemanticVersions(cv.Status.AvailableUpdates)
-		for _, update := range cv.Status.AvailableUpdates {
-			fmt.Fprintf(w, "  %s\t%s\n", update.Version, update.Image)
+	majorMinorBuckets := map[uint64]map[uint64][]configv1.ConditionalUpdate{}
+
+	for i, update := range cv.Status.ConditionalUpdates {
+		version, err := semver.Parse(update.Release.Version)
+		if err != nil {
+			fmt.Fprintf(o.ErrOut, "warning: Cannot parse SemVer available update %q: %v", update.Release.Version, err)
```

Comment:
We should `continue` here because `version` content is not reliable

Reply:
I've pushed 46d8a1c096dc -> 07d7b03745e5b3, which adds this `continue`.